### DISCUSSION TITLE
`chapter_ranges`'s alpha

### DISCUSSION THREAD

**Discussion by hooke007** [tomasklaen/uosc#337] [General]
Hello tomasklaen, I found that the opacity of `chapter_ranges` is not controlled by timeline's opacity, is it a intended design?

**Comment by tomasklaen** [comment]
They have their own opacity control, as their color accepts rgb(a) hex values: `XXYYZZ80` -> `0.5` opacity.

**Reply by hooke007** [reply]
Yeah I know, so it is not a bug? I used to think it ought to follow the timeline's opacity rather than controlled by another option.

**Reply by tomasklaen** [reply]
No. It's more configurable this way. You can have lower timeline opacity and higher chapter range opacity. I don't know why would someone want that, but it's an option now and wouldn't be if range's opacity was multiplied by timeline's.

**Reply by hooke007** [reply]
> lower timeline opacity and higher chapter range opacity

Actucally it was the cause I found it strange for me. In this case the current-time indicator is not clear.

https://user-images.githubusercontent.com/41094733/197330628-577cb812-502d-422c-9553-f5f200e5c516.mp4

**Reply by tomasklaen** [reply]
You've increased the opacity of that range so you can't see through it, and now you can't see through it. You've just configured your uosc out of being usable. I don't understand what you want from me here 😞.

**Reply by hooke007** [reply]
Maybe I cannot get the point of this design... Never mind, I seldom change those options just report my findings.
